readme.md

```diff
@@ -176,74 +186,78 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 ---
 
 - [ ] Update one of the `POST` routes you've created by adding the following code before the response is set. Then, try sending a request body using Postman to that route.
   ```js
-  console.log(req.body)
+  console.log(req.body);
   ```
 
 * **Question:** Try commenting out `body-parser` in your application. What happens and why?
 
-* **Your Answer:**
+* **Your Answer:** We get an error stating `Cannot destructure property 'content' of 'undefind' or 'null'`. This happens because `body-parser` gives us access to `req.body`. By commenting it out, `body-parser` doesn't know what to assume about the request coming in, in this example, a JSON object and so `content` comes back as `undefined`.
 
 ---
 
 - [ ] Visit the [Exercise: Express](https://github.com/pce-uw-jscript400/exercise-express) repository on GitHub. Follow the setup instructions and open the code in a code editor. Then, answer the following questions:
 
 * **Question:** Describe what is happening on the following line. What is the [shortid](https://www.npmjs.com/package/shortid) package and what is it being used for?
+
   ```js
-  const { generate: generateId } = require('shortid')
+  const { generate: generateId } = require("shortid");
   ```
 
-* **Your Answer:**
+* **Your Answer:** ShortId generates unique and short ids that are URL friendly and that's how we seem to be using it inside the app.
+
+- **Question:** What routes are currently available to you as part of this application?
 
-* **Question:** What routes are currently available to you as part of this application?
+- **Your Answer:**
 
-* **Your Answer:**
+* `/vegetables`
+* `/vegetables/:id`
 
-* **Question:** Look for `helpers.validate` in the `app.js` file. What is this and how does it work?
+- **Question:** Look for `helpers.validate` in the `app.js` file. What is this and how does it work?
 
-* **Your Answer:**
+- **Your Answer:** `helpers.validate` is a custom middleware that seems to be checking to make sure that the `req.body` contains valid information on the `POST` request. If the `req.body` is empty, it returns an error. If the `req.body` does not have all the required keys for `name` and `price`, then it will return an error. It also checks to make sure that there are no extra keys, in which case it returns an error.
 
-* **Question:** Try creating a new vegetable. Then, try restarting your server. What happens to the data you posted and why?
+- **Question:** Try creating a new vegetable. Then, try restarting your server. What happens to the data you posted and why?
 
-* **Your Answer:**
+- **Your Answer:** My data did not persist. This happened because as of now, it's just using the HTTP `POST` method to send data to a route and we are not using a database or a JSON file to store the contents that are being created.
```

Comment:
Correct, the data is being stored as an object which does not persist once the server is restarted.